cli/src/main/java/org/infinispan/cli/completers/ConnectorCompleter.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.cli.completers;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.infinispan.cli.Context;
+
+/**
+ * A {@link org.aesh.command.completer.OptionCompleter} for protocol connectors.
+ *
+ * @author Tristan Tarrant
+ * @since 12.1
+ */
+public class ConnectorCompleter extends ListCompleter {
+   @Override
+   Collection<String> getAvailableItems(Context context) throws IOException {
+      return context.getConnection().getConnectorNames();
```

Comment:
```suggestion
      try {
         return context.getConnection().getConnectorNames();
      } catch (Exception e) {
         return Collections.emptyList();
      }
```

I would put a try-catch block to avoid adding the exception stack trace to the console. Example: 
```
[pedro-laptop-3-52395@cluster//containers/default]> server connector ipfilter ls java.nio.file.AccessDeniedException: The user is not allowed to access the server resource: ISPN000287: Unauthorized access: subject 'Subject with principal(s): [pedro, RolePrincipal{name='reader'}]' lacks 'ADMIN' permission
	at org.infinispan.cli.connection.rest.RestConnection.handleResponseStatus(RestConnection.java:163)
	at org.infinispan.cli.connection.rest.RestConnection.parseBody(RestConnection.java:129)
	at org.infinispan.cli.connection.rest.RestConnection.getConnectorNames(RestConnection.java:372)
```